backend/src/test/java/com/morak/back/team/acceptance/TeamAcceptanceTest.java

```diff
@@ -107,52 +172,95 @@ public void setUp() {
     @Test
     void 그룹에_참가한_멤버의_그룹_참가_여부를_확인한다() {
         // given
-        String teamLocation = 기본_그룹_생성을_요청한다().header("Location");
-        String teamInvitationLocation = 그룹_초대코드_생성을_요청한다(teamLocation).header("Location");
+        TeamCreateRequest teamCreateRequest = new TeamCreateRequest("하이");
+        String teamLocation = 그룹_생성을_요청한다(teamCreateRequest, token).header("Location");
+        String teamInvitationLocation = 그룹_초대코드_생성을_요청한다(teamLocation, token).header("Location");
 
         // when
         ExtractableResponse<Response> response = 그룹_참가_여부_조회를_요청한다(teamInvitationLocation, token);
-        InvitationJoinedResponse isJoinedResponse = response.as(InvitationJoinedResponse.class);
 
         // then
+        InvitationJoinedResponse invitationJoinedResponse = response.as(InvitationJoinedResponse.class);
         Assertions.assertAll(
                 () -> assertThat(response.statusCode()).isEqualTo(HttpStatus.OK.value()),
-                () -> assertThat(isJoinedResponse.getGroupCode()).hasSize(8),
-                () -> assertThat(isJoinedResponse.getName()).isEqualTo("albur"),
-                () -> assertThat(isJoinedResponse.getIsJoined()).isTrue()
+                () -> assertThat(invitationJoinedResponse)
+                        .usingRecursiveComparison()
+                        .isEqualTo(new InvitationJoinedResponse(teamLocation.split("/")[3], teamCreateRequest.getName(),
+                                true))
```

Comment:
```suggestion
                        .isEqualTo(new InvitationJoinedResponse(getTeamCodeFromLocation(teamLocation), teamCreateRequest.getName(),
                                true))
```
R: 이 부분이 7번이나 중복되고 있어서 메서드로 빼는게 좋을 것 같아요. 이름은 `getTeamCode()`나 `extractTeamCodeFromLocation()` 등등 까라가 원하는걸로..ㅎㅎ
PollAcceptanceTest에도 적용하면 좋을 것 같아요~

Reply:
반영했습니다 엘리 ~ 예리하시네요 ! 
중복은 못참죠 !